### PR TITLE
CLD-5271 Add changes necessary for Close monitoring - support for red…

### PR DESCRIPTION
**Pull request by mpivarnikohpen**
…irects and make local script optional

### CODE REVIEW

File: modules/multi_step/variables.tf [hunk 1]
```diff
@@ -83,6 +83,13 @@ variable "script_relative_path" {
   description = "Relative path to file from current working directory. Incl. filename"
   default     = ""
 }
+
+variable "create_local_script" {
+  description = "Create local test script"
+  type = string
+  default = "true"
```
Comment (lucky-edu) [inline modules/multi_step/variables.tf:90]: use boolean type

**Review by lucky-edu — CHANGES_REQUESTED** [review]

File: modules/multi_step/template.tf [hunk 2]
```diff
@@ -1,4 +1,5 @@
 resource "local_file" "run_test_script" {
+  count               = var.create_local_script == "true" ? 1 : 0
```
Comment (jstehlik-ohpen) [inline modules/multi_step/template.tf:2]: formatting

File: modules/multi_step/variables.tf (hunk 1 above)
Comment (jstehlik-ohpen) [reply modules/multi_step/variables.tf:90]: should default be true?

**Comment by lucky-edu** [comment]
seems you are creating a pr from a fork. Any reason to do that?
Could you simply clone the repo?

**Pull request closed without merging**